.github/workflows/test-contract.yaml

```diff
@@ -0,0 +1,35 @@
+name: tests-contracts
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  tests-contracts:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: docker/setup-buildx-action@v3
+      - name: Check out to finschia-wasm
+        uses: actions/checkout@v4
+        with:
+          repository: Finschia/finschia-wasm
+
+      - name: Update cargo.lock
+        run: |
+          cargo build
+        working-directory: examples
+
```

Comment:
Is this needed?

Reply:
Yes. The "cargo.lock" file has not been uploaded, so it is needed to generate.

Reply:
I uploaded "Cargo.lock" and this step is removed.